docs/sources/configuration/examples.md

```diff
@@ -32,6 +32,32 @@ schema_config:
 ```
 
 
+## 10-S3-And-DynamoDB-With-KMS-Snippet.yaml
```

Comment:
Since this is numbered as "10" is there some reason it is so early in the file and doesn't come after "9"?

Reply:
This file is auto generated by `make generate-example-config-doc` and checked-in as-is.  There was a pipeline step that failed that provided instructions specifically to run that command and checkin the file.  My guess is the sort order isn't treating the first field as a number.

Reply:
FYI - I cleaned up the `generate-example-config-doc` target in the Makefile.  This now sorts by numerical ordering, and I think its quite a bit more readable now too.